stdlib/public/Concurrency/Task.swift

```diff
@@ -850,6 +862,18 @@ func _taskCancel(_ task: Builtin.NativeObject)
 @usableFromInline
 func _taskIsCancelled(_ task: Builtin.NativeObject) -> Bool
 
+// TODO (rokhinip) : This availability probably ought to change? Or we need to do something here
+@available(SwiftStdlib 5.1, *)
```

Comment:
I think this should be 9999 for the moment, or 5.6 if this is also going on the 5.6 branch (I assume it isn't).

Reply:
To be fully correct, yeah, they're only available in the future.  But as long as they're just implementation details and not directly exposed, it doesn't matter.  Just remove `@usableFromInline` and be explicit about them being `internal`.

Reply:
Oh yeah, `internal` is the way to go if we don't need it exposed.